src/main/java/at/aau/se2/handler/game/subhandlers/SwitchCardDeckHandler.java

```diff
@@ -19,6 +19,8 @@ public void handleMessage(WebSocketSession session, JsonNode msg, Lobby lobby){
             String[] infos = readInfosFromMessage(msg);
             List<Actioncard> cards = UtilityMethods.findPlayer(session, lobby).getCards();
             Actioncard newCard = drawRandomCard(lobby);
+            String test = infos[1];
+            System.out.println("Test in der handlemessage"+infos[1]);
```

Comment:
`System.out.println()` nicht verwenden
```suggestion
            Logger.getLogger("global").info("Test in der handlemessage"+infos[1]);
```